modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -234,6 +237,9 @@ TensorMechanicsApp::registerObjects(Factory & factory)
   registerAux(NewmarkVelAux);
 
   registerBoundaryCondition(Pressure);
+
+  registerPostprocessor(Mass);
```

Comment:
Wow I can react to my own comment. I guess if I'm torn on something I can give myself a thumbs up and down.